### PR TITLE
Removed gradle 'application' plugin from the application template

### PR DESCRIPTION
**Pull request by gsartori**
The Gradle "application" plugin is not required by a Spring Boot application, it was introduced in the Grails app template with the inclusion of Micronaut.

From the Gradle user guide:

> The Application plugin facilitates creating an executable JVM application. It makes it easy to start the application locally during development, and to package the application as a TAR and/or ZIP including operating system specific start scripts.

I propose to remove the plugin, as part of a cleanup effort, since this is not the way we are using it in a Grails application.

### CODE REVIEW

**Comment by jamesfredley** [comment]
@gsartori You will also need to update `GrailsApplicationSpec` and then the build should be successful.

**Comment by gsartori** [comment]
Thank you for the hint, it should be okay now

File: grails-forge-core/src/test/groovy/org/grails/forge/feature/lang/GrailsApplicationSpec.groovy [hunk 1]
```diff
@@ -24,12 +24,6 @@ class GrailsApplicationSpec extends BeanContextSpec implements CommandOutputFixt
         applicationGroovyFile.contains("@CompileStatic")
         !applicationGroovyFile.contains("@PluginSource")
 
-        when:
-        def buildGradle = output['build.gradle']
-
-        then:
-        buildGradle.contains('mainClass.set("example.grails.Application")')
-
         where:
```
Comment (matrei) [inline grails-forge-core/src/test/groovy/org/grails/forge/feature/lang/GrailsApplicationSpec.groovy:27]: Should the test name be updated to reflect that the `Application` class no longer is referenced in `build.gradle`?

**Review by matrei — APPROVED** [review]

File: grails-forge-core/src/test/groovy/org/grails/forge/feature/lang/GrailsApplicationSpec.groovy (hunk 1 above)
Comment (gsartori) [reply grails-forge-core/src/test/groovy/org/grails/forge/feature/lang/GrailsApplicationSpec.groovy:27]: I guess so, I've updated it

**Review by jamesfredley — APPROVED** [review]

**Pull request merged**